tensorflow/core/kernels/deep_conv2d.cc

```diff
@@ -1,11 +1,8 @@
 /* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
-
```

Comment:
Unrelated changes and also illegal ones. Copyright notices should not be altered